src/NoteParser.ts

```diff
@@ -177,6 +186,17 @@ export class Note {
       });
     return _tagSet;
   }
+
+  // completionItem.documentation ()
+  documentation(): string | vscode.MarkdownString | undefined {
+    if (this.data === undefined) {
+      return "";
+    } else if (NoteWorkspace.compileSuggestionDetails()) {
+      return new vscode.MarkdownString(this.data);
```

Comment:
Do we need to wrap the call to `MarkdownString` in a try catch? Could it raise an exception for any weird string value of `this.data` ?

Reply:
Actually I'm not sure if an exception will be raised, but adding a `try catch` shouldn't be a problem and I'll fix it.